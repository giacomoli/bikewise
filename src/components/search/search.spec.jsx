import React from 'react';
import ReactDOM from 'react-dom';
import { mount } from 'enzyme';
import { TextField } from '@material-ui/core';

import Search from './index';

const deepContainer = mount(<Search />);

describe('<Search /> with no props', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Search />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('Search has three Search Fields', () => {
    expect(deepContainer.find(TextField)).toHaveLength(3);
  });
});
