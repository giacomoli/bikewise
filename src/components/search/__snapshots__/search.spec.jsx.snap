// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Search /> with no props should match the snapshot 1`] = `ShallowWrapper {}`;
